preload/data/songs/eggnog/eggnog-metadata-erect.json

```
@@ -2,33 +2,72 @@
   "version": "2.2.4",
   "songName": "Eggnog Erect",
   "artist": "Saruky",
```

```suggestion
  "artist": "Saruky",
  "charter": "fabs + SpazKid",
```

Heads up, exporting charts removes the charter from the Metadata!